### DISCUSSION TITLE
no bpool

### DISCUSSION THREAD

**Discussion by steinpath** [Sithuk/ubuntu-server-zfsbootmenu#42] [General]
may I ask one thing..? why have you not considerered a bpool partition for efi/boot in your script? there is one in every project/script on ubuntu/zfs, and its absence entails compatibility problems.
anyway the script, and the system, works..

**Comment by Sithuk** [comment]
The thread below and in particular the comment from ahesford, one of the zfsbootmenu authors, should be helpful in answering your question.
https://www.reddit.com/r/zfs/comments/zlmpww/comment/j06f7yn/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

**Reply by steinpath** [reply]
Thank you!
